wolfcrypt/src/asn.c

```diff
@@ -4236,6 +4252,75 @@ static int GetName(DecodedCert* cert, int nameType)
 
             cert->srcIdx += strLen;
         }
+    #ifdef WOLFSSL_CERT_EXT
+        else if ((0 == memcmp(&cert->source[cert->srcIdx],
+                    "\x2b\x06\x01\x04\x01\x82\x37\x3c\x02\x01", 10)) &&
+                 ((cert->source[cert->srcIdx + 10] == 0x3) ||
+                  (cert->source[cert->srcIdx + 10] == 0x2)))
```

Comment:
Can you add a define for these consts so its easier to understand? Including the places 0x03 and 0x02 are used in the if logic below.

Reply:
Got it